manifest.json

```diff
@@ -987,6 +987,50 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm",
+              "dockerfile": "src/ubuntu/21.04/helix/arm32v7",
+              "os": "linux",
+              "osVersion": "bionic",
```

Comment:
This should be `hirsute` instead of `bionic`. Also applies to the other cases.  This requires an update to the infrastructure (sending you a PR).